### ISSUE TITLE
build: consider setting gopherjs build tag on default

### ISSUE THREAD

**Issue by paralin** [gopherjs/gopherjs#1046]
At the moment "js" is set as a tag, but "gopherjs" is not.

Consider setting "gopherjs" as a default tag to distinguish between Wasm and GopherJS environments.

**Comment by nevkontakte** [comment]
Agreed, this is something I have thought of doing a few times as well, but never gotten around to it. Would you like sending a pull request?
